docs/ref-apack.md

```diff
@@ -0,0 +1,21 @@
+---
+title: APACK
+category: reference
+order: 80
+---
+
+Packages with a class that implements `ZIF_APACK_MANIFEST` (interface is automatically created at initial start of abapGit) can define metadata that is interpreted by clients which support the APACK package and dependency management framework. Such an implementation class must reside in the top-level ABAP package which is serialized by abapGit so that it can be detected properly. After the class is instantiated, it needs to have the field `ZIF_APACK_MANIFEST~DESCRIPTOR` populated with the respective APACK metadata. The metadata follows the (https://maven.apache.org/pom.html#Maven_Coordinates)[POM format by Apache Maven] and specifies additional information as well:
+- `group_id`: Unique name for your organization or project, e.g. `github.com/larshp`
+- `artifact_id`: Name for the project, needs to be unique together with the specified `group_id`, e.g. `abapGit`
+- `version`: Version of your project, should adhere to the (https://semver.org/)[semantic versioning concept], e.g. `1.85`
+- `repository_type`: Currently fixed to `abapGit` as it's currently the only supported repository type and is set automatically during serialization
+- `git_url`: The URL where your main repository is located, e.g. `https://github.com/larshp/abapGit.git`
+- `dependencies`: Optional, only needs to be filled if this project has dependencies to other projects. Be sure to specify the `git_url` for each dependency properly as this is currently the only way how dependencies can automatically be resolved.
```

Comment:
What exactly is the format of `dependencies`?

Reply:
I added some additional information with my latest commit. :)

Reply:
Thank you! It was not clear to me. I am not familiar with maven.